osm2streets/src/transform/separate_cycletracks.rs

```diff
@@ -103,6 +103,8 @@ fn snap(streets: &mut StreetNetwork, input: Cycleway) {
     // Remove the cycleway, but remember the lanes it contained
     let mut cycleway_lanes = streets.remove_road(input.cycleway).lane_specs_ltr;
 
+    // TODO Preserve osm_ids
```

Comment:
Thought of this just now, unrelated to this PR. I'm going to be picking up work here after the refactor dust settles.